terraform/modules/signon_bearer_token/iam.tf

```diff
@@ -113,3 +113,28 @@ resource "aws_iam_policy" "vpc" {
     ]
   })
 }
+
+resource "aws_iam_role_policy_attachment" "s3" {
+  role       = aws_iam_role.lambda_execution_role.name
+  policy_arn = aws_iam_policy.s3.arn
+}
+
+resource "aws_iam_policy" "s3" {
+  name        = "${local.secret_name}_s3_write"
+  path        = "/"
+  description = "Allow lambda to write events to S3 deploy-events bucket"
+
+  policy = jsonencode({
+    Version = "2012-10-17",
+    Statement = [
+      {
+        Action = [
+          "s3:PutObject",
+          "s3:PutObjectAcl"
```

Comment:
Just wondering what `PubObjectAcl` is needed for specifically?

Reply:
This is a [required permission](https://github.com/concourse/s3-resource#required-iam-permissions) for the concourse resource. 

When putting an object you can set the ACL on it, I think the default is [`private`](https://docs.aws.amazon.com/AmazonS3/latest/userguide/acl-overview.html#canned-acl).

```
- put: release
  params:
    file: path/to/release-*.tgz
    acl: public-read
```

Reply:
Interesting, thanks!